department-of-veterans-affairs/abd-vro
# MCP-2087: Add date and time to python logging messages

## What was the problem?

No way to see when messaged were processed by python condition service. 
Associated tickets or Slack threads:

- [MCP-2087](https://amida.atlassian.net/browse/MCP-2087)

## How does this fix it?[^1]

Adds date and time to logging
## How to test this PR
- Step 1 Build and run app.
- Step 2 Send request with full-health endpoint. Looks at docker logs and see that message has a date and time. 
- [^secrel]


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
